authbridge/demos/github-issue/aiac/aiac_agent/utils/validators.py

```
@@ -0,0 +1,291 @@
+        if verbose or not roles_correct or not mappings_correct:
+            print("\n" + "=" * 80)
+            print("POLICY VERIFICATION:")
+            print("=" * 80)
+            print(f"Roles Correct: {'YES' if roles_correct else 'NO'}")
+            print(f"Mappings Correct: {'YES' if mappings_correct else 'NO'}")
+            print(f"Explanation: {explanation}")
+            print("=" * 80 + "\n")
+
+        return (roles_correct, mappings_correct, explanation)
+
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)